dist/stat_data.py

```diff
@@ -727,6 +727,8 @@ def __init__(self, name, desc, flags=''):
     # Compression statistics
     ##########################################
     CompressStat('compress_read', 'compressed pages read'),
+    CompressStat('compress_precomp_intl_max_page_size', 'compressed page maximum internal page size prior to compression', 'no_clear,no_scale,size'),
```

Comment:
I think "compressed internal page maximum size prior to compression" or "compressed maximum internal page size prior to compression" sounds better.  If changing please change the next one as well accordingly.